app/views/homepage/_links_and_search.html.erb

```diff
@@ -19,6 +19,13 @@
                   track_value: 1,
                   track_dimension_index: 29,
                   track_dimension: item[:text],
+                  ga4_link: {
+                    'event_name': 'navigation',
+                    'type': 'home page',
+                    'index': "#{index}",
+                    'index_total': "#{t("homepage.index.popular_links").length}",
```

Comment:
Could this value be pre-calculated rather than once through every pass of the loop?

Reply:
Thank you good spot, have updated